pkg/storkctl/snapshot_test.go

```diff
@@ -50,12 +52,17 @@ func testSnapshotsCommon(t *testing.T, newCommand NewTestCommand, cmdArgs []stri
 	cmd := newCommand(f, streams)
 	cmd.SetOutput(buf)
 	cmd.SetArgs(cmdArgs)
-	if err = cmd.Execute(); err != nil {
-		require.NoError(t, err, "Error executing command: %v", cmd)
-	}
 
-	if e, a := expected, buf.String(); e != a {
-		t.Errorf("expected %v, got %v", e, a)
+	err = cmd.Execute()
+
+	if !behaviorOnFatal {
+		if err != nil {
```

Comment:
No need of the if, require.NoError() checks that

Reply:
Fixed